### ISSUE TITLE
Text multiline item starts with value of a single space

### ISSUE THREAD

**Issue by pbowen-oc** [OpenClinica/enketo-express-oc#203]
When I open a new form with a text multiline item, the item starts with the value of a single space. If I don't enter a value for this item and I reopen the form as an existing record, it still appears with the value of a space.

When I start typing in this item, my text automatically begins after the space, so a string length constraint I apply to the item does not produce the expected results.

![2019-04-23 19_36_45-Window](https://user-images.githubusercontent.com/25179620/56622789-783f7000-65ff-11e9-8107-d0150017d152.png)

**Comment by MartijnR** [comment]
right, this is by design, but I don't remember what for. Will revisit and see if we can finally remove that weird need for a space!
